src/core/heila-core.js

```diff
@@ -6,6 +6,38 @@ const BPromise = require('bluebird');
 const {knex} = require('../util/database').connect();
 
 
+function checkHeilaTableStatus(user) {
+  return findByUuid(user.uuid)
+    .then(foundHeila => {
+      
+      // if heila table row doesn't exist -> create it
+      if (foundHeila === null && user.heila === true) {
+        console.log('heila not found + heila = true --> add table row');
```

Comment:
Remove debug logging

Reply:
Ha! I shall keep my debug logging until we ship the final product. Everything will be removed at that time. But my workflow suffers if I remove all of it :/